backend/api.py

```diff
@@ -102,5 +99,5 @@ def get_any_deal(deal):
 output["daydeal_weekly"] = deals[2]
 output["blickdeal"] = deals[3]
 output["galaxus"] = deals[4]
-with open("/var/www/alldeals/frontend/deals.json", "w") as f:
+with open("/var/www/alldeals-beta/frontend/deals.json", "w") as f:
```

Comment:
Maybe we should make this relative and/or separate frontend and backend again

Reply:
I would be in favor of separating frontend and backend more clearly again. I am not a big fan of the data provider updating the frontend. Having the backend providing the data via API and the frontend consuming the data from the API as originallywoulf be more layered architecture compliant.

Reply:
Yes, let's create an issue for that and work on it at some point.